[BUG] Failed to create fedora spot instance using mapt

```
podman run -d  --name create-fedora \
            -v ${PWD}:/workspace:z \
            -e AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} \
            -e AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} \
            -e AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION} \
            quay.io/redhat-developer/mapt:v0.8.0-dev-linux aws fedora create \
                --project-name fedora \
                --backed-url "file:///workspace" \
                --conn-details-output "/workspace" \
                --tags project=crc,environment=local,user=prkumar \
                --spot \
                --airgap \
                --cpus 6 \
                --nested-virt \
                --memory 16


DEBU running 'mapt aws fedora create'             
DEBU context initialized for mapte6ae042d         
DEBU checking stack spotOption-fedora             
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/redhat-developer/mapt/pkg/provider/aws/modules/spot.getOutputs(0xc0007e0240)
	/workspace/pkg/provider/aws/modules/spot/stack.go:137 +0x3a5
github.com/redhat-developer/mapt/pkg/provider/aws/modules/spot.SpotOptionRequest.Create({{0x556e642, 0x4}, {0x5577b19, 0xa}, {0xc0005df710, 0x3, 0x3}, {0xc0022de060, 0x26}, {0x557063f, ...}})
	/workspace/pkg/provider/aws/modules/spot/stack.go:69 +0x2b4
github.com/redhat-developer/mapt/pkg/provider/aws/action/fedora.Create(0xc000532140)
	/workspace/pkg/provider/aws/action/fedora/fedora.go:87 +0x718
github.com/redhat-developer/mapt/cmd/mapt/cmd/aws/hosts.getFedoraCreate.func1(0xc000005b00, {0xc0004b43c0, 0x0, 0xf})
	/workspace/cmd/mapt/cmd/aws/hosts/fedora.go:74 +0x77b
github.com/redhat-developer/mapt/cmd/mapt/cmd.executeWithLogging.func1(0xc000005b00, {0xc0004b43c0, 0x0, 0xf})
	/workspace/cmd/mapt/cmd/root.go:93 +0x123
github.com/spf13/cobra.(*Command).execute(0xc000005b00, {0xc0004b42d0, 0xf, 0xf})
	/workspace/vendor/github.com/spf13/cobra/command.go:983 +0xfbb
github.com/spf13/cobra.(*Command).ExecuteC(0x8985f60)
	/workspace/vendor/github.com/spf13/cobra/command.go:1115 +0xa2e
github.com/spf13/cobra.(*Command).Execute(0x8985f60)
	/workspace/vendor/github.com/spf13/cobra/command.go:1039 +0x32
github.com/spf13/cobra.(*Command).ExecuteContext(0x8985f60, {0x5acb390, 0x8aa93a0})
	/workspace/vendor/github.com/spf13/cobra/command.go:1032 +0x73
github.com/redhat-developer/mapt/cmd/mapt/cmd.Execute()
	/workspace/cmd/mapt/cmd/root.go:70 +0x74
main.main()
	/workspace/cmd/mapt/main.go:6 +0xf
```

--- a/pkg/provider/aws/modules/spot/stack.go
+++ b/pkg/provider/aws/modules/spot/stack.go
@@ -1,6 +1,8 @@
 package spot
 
 import (
+	"errors"
+
 	"github.com/pulumi/pulumi/sdk/v3/go/auto"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 	"github.com/redhat-developer/mapt/pkg/manager"
@@ -132,6 +134,9 @@ func getOutputs(stack *auto.Stack) (*SpotOptionResult, error) {
 	outputs, err := manager.GetOutputs(stack)
 	if err != nil {
 		return nil, err
+	}
+	if len(outputs) == 0 {
+		return nil, errors.New("Stack outputs are empty please destroy and re-create")
 	}
 	return &SpotOptionResult{
 		Region:           outputs["region"].Value.(string),

